services/spar/src/Spar/API/Swagger.hs

```diff
@@ -156,3 +149,26 @@ instance ToParamSchema BindCookie where
 
 instance ToSchema Void where
   declareNamedSchema _ = declareNamedSchema (Proxy @String)
+
+instance ToParamSchema ScimToken where
+  toParamSchema _ = toParamSchema (Proxy @Text)
+
+instance ToSchema ScimToken where
+  declareNamedSchema _ = declareNamedSchema (Proxy @Text)
+    & mapped . schema . description ?~ "Authentication token"
+
+instance ToSchema ScimTokenInfo where
```

Comment:
Should I create Spar.SCIM.Swagger and move these there?

Reply:
Yeah, I think I should.

Reply:
I have no idea which would be better.  It's nice to have the docs in one place, but it's also nice to have the docs in the place where the code is.  Your call!